leavestracker/celery.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import, unicode_literals
+from celery import Celery
+from celery.schedules import crontab
+from datetime import datetime, timedelta
+
+import os
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "leavestracker.settings.local")
+
+app = Celery('leavestracker')
+
+app.config_from_object('django.conf:settings', namespace='CELERY')
+
+app.conf.beat_schedule = {
+    'add-every-10-seconds': {
+        'task': 'leavestracker.apps.leaves.tasks.send_notification',
+        'schedule': crontab(minute=42 , hour=17),
```

Comment:
we need to add it to 9 am hence minute=0, and hour=9. why we have it for 42 and 17 ?